app/scripts/components/common/mapbox/layer-legend.tsx

```diff
@@ -35,9 +35,15 @@ type LegendSwatchProps = {
 };
 
 const makeGradient = (stops: string[]) => {
-  const d = 100 / stops.length - 1;
-  const steps = stops.map((s, i) => `${s} ${i * d}%`);
-  return `linear-gradient(to right, ${steps.join(', ')})`;
+  const cCount = stops.length;
+  if (cCount === 1) {
+    return stops[0];
+  } else {
+    const [first, ...others] = stops;
```

Comment:
I think this approach is fine, but does this do anything different from adding parenthesis to the original code? like 
```
  if (stops.length === 1 ) return stops[0];
  const d =  100 / (stops.length - 1);
  const steps = stops.map((s, i) => `${s} ${i * d}%`);
  return `linear-gradient(to right, ${steps.join(', ')})`;
 ```

Reply:
Oh wow. You're absolutely right. 😅
Talk about overly complicating things. Thank you